cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,16 @@
+import time
+
+from django.core.management.base import BaseCommand
+from django.db import connections, OperationalError
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **kwargs):
+        while True:
+            try:
+                connections["default"].ensure_connection()
+                break
+            except OperationalError:
+                self.stdout.write("Server loading...")
```

Comment:
Consider using `self.stdout.write(self.style.WARNING('Server loading...'))` to provide a more visually distinct warning message to the console.